ListSum/ListSum.py

```diff
@@ -0,0 +1,54 @@
+def get_list_one_by_one(lst):
+    """ get list of numbers, strings are ignored. return: list of numbers """
+    print('Type a number, type "stop" to end the list')
+    member = input()
+    if(member == "stop"):
+        return lst
+    else:
+        if(member.isdigit()):
+            lst.append(int(member))
+        else:
+            print("{} is ignored, not a number".format(member))
+        return get_list_one_by_one(lst)
+
+
+def get_full_list(lst):
+    """ get list of numbers, if typed not a number, user must retype. return: list of numbers """
+    while True:
+        print("Type values seperated by a comma")
+        list_as_str = input()
+        lst = (list_as_str.split(","))
+        # Convert list members to integer
+        try:
```

Comment:
using try,except block is fine here, but we should try using it only when necessary. try doing what you are doing without except ( hint: read about `filter`).